src/components/Request/MethodField.js

```diff
@@ -1,29 +1,33 @@
 import React, { PropTypes } from 'react';
 import { connect } from 'react-redux';
-import { Col, Row, FormGroup, FormControl, ControlLabel, Button } from 'react-bootstrap';
+import { Col, Row, FormGroup, FormControl, ControlLabel } from 'react-bootstrap';
 
 import * as Actions from 'store/config/actions';
 import { isEditMode } from 'store/config/selectors';
 import { REQUEST_METHODS } from 'constants/constants';
 
-function MethodField({ input, meta, editMode }) {
-  const isCustom = !REQUEST_METHODS.slice(0, -1).includes(input.value);
+export function checkIfCustom(value) {
+  return !REQUEST_METHODS.slice(0, -1).includes(value);
+}
+function MethodField({ input, meta }) {
+  const isCustom = checkIfCustom(input.value);
 
   return (
     <FormGroup
       controlId="method"
       validationState={meta.invalid ? 'error' : undefined}
     >
-      <Col
-        componentClass={ControlLabel}
-        sm={2}
-      >
-        Method
-      </Col>
 
-      <Col sm={7}>
+      <Col sm={12}>
         <Row>
           <Col xs={isCustom ? 6 : 12}>
+
+            <ControlLabel
+              bsClass="pseudo-hidden"
```

Comment:
Awesome that you think about usability and add a hidden label!

One minor thing:

In bootstrap you can use the `sr-only` (screen reader-only) class to make it hidden from view. Try using that instead of the `psuedo-hidden` class